### ISSUE TITLE
storage_state call resize browser window

### ISSUE THREAD

**Issue by algot** [microsoft/playwright-python#736]
**Context:**

- Playwright Version: 1.11.1
- Operating System: Ubuntu 20.04
- Python Version: 3.9.5

test script
```
from playwright.sync_api import Page


def test_issue(page: Page):
    page.goto('http://playwright.dev')
    page.wait_for_timeout(1000)
    page.context.storage_state()
    page.wait_for_timeout(1000)
```
conftest.py
```
from playwright.sync_api import sync_playwright
from playwright.sync_api import Page
import pytest


@pytest.fixture(scope='function')
def page():
    with sync_playwright() as p:
        browser = p.chromium.launch(headless=False)
        new_page: Page = browser.new_page()
        new_page.set_viewport_size({'width': 1848, 'height': 1080})

        yield new_page

        browser.close()
```

Description:
context.storage_state() call resize browser window but leave viewport size untouched

Why browser size changes and how to prevent it?

See video capture - https://i.imgur.com/vNAUSkF.gif

**Comment by mxschmitt** [comment]
This is probably related to https://github.com/microsoft/playwright/issues/6743

@yury-s wdyt?

**Comment by yury-s** [comment]
Sounds relevant, I wonder if it reproduces if you replace  `page.context.storage_state()` with

```
page2 = page.context.new_page()
page2.goto(...)
```

**Comment by algot** [comment]
> 
> 
> Sounds relevant, I wonder if it reproduces if you replace `page.context.storage_state()` with
> 
> ```
> page2 = page.context.new_page()
> page2.goto(...)
> page2.close()
> ```

Error occurs

My code:
```
def test_issue(page: Page):
    page.goto('http://playwright.dev')
    page.wait_for_timeout(1000)
    page2 = page.context.new_page()
    page2.goto(...)
    page.wait_for_timeout(1000)
    page2.close()
```

```
    async def new_page(self) -> Page:
        if self._owner_page:
>           raise Error("Please use browser.new_context()")
E           playwright._impl._api_types.Error: Please use browser.new_context()

.venv/lib/python3.9/site-packages/playwright/_impl/_browser_context.py:138: Error
```

**Comment by mxschmitt** [comment]
@algot I verified and Yurys assumption is correct. So I will close this as a duplicate of the upstream bug.

**Comment by algot** [comment]
@mxschmitt 
I recheck the issue on the latest release which looks like should be fixed in 1.12.1
Issue is still reproduced with the test from my initial post.
Please check

**Comment by mxschmitt** [comment]
The upstream bug is still open: https://github.com/microsoft/playwright/issues/6743

We try to come up with a fix for that in 1.13. Since this is an upstream bug I closed the issue here.

**Comment by algot** [comment]
@mxschmitt 
Oh, sorry. I saw that #7058 was fixed and was confused
Sorry for the trouble
